lib/ansible/modules/apt.py

```diff
@@ -690,28 +690,44 @@ def parse_diff(output):
     return {'prepared': '\n'.join(diff[diff_start:diff_end])}
 
 
-def mark_installed_manually(m, packages):
+def mark_installed(m, packages, state="manual"):
```

Comment:
Since `state` is just a boolean, change it to `manual: bool` or `auto: bool`. That will simplify the conditional below and eliminate the unnecessary else case.

Reply:
Thanks for the suggestion.